cli/auth.go

```diff
@@ -129,15 +129,15 @@ func (c *viamClient) loginAction(cCtx *cli.Context) error {
 		already := "Already l"
 		if !alreadyLoggedIn {
 			already = "L"
-			viamLogo(cCtx.App.Writer)
+			viamLogo(cCtx.App.ErrWriter)
```

Comment:
This prints:
```

@@BO..    "%@@B^%@@<      .}j.      !B@B$v'.    'nB$$$!
.*@$%l   f$$@X  %$$+      &@$$      l$$$$@@^   "B$$$$@!
  (@$$0'M@$@:   B$$~    ~B$$@$@1    !$$BQ@@@q.0@$$0@$$!
   'WB$$B@p     B$$~   qB$%-!@@@o.  l$@B..B@$@$@%; @$@!
    .u$$$!      B$$~ ;@@@& ... oB$@~!!$$@. z$$$z'. $$$!
      :h'       B$$~L%@$||     -@@$bi$@B    'M'    $$$!
```
Which is very cool and adds lots of personality. 

However, it also shows up when logging in via an api-key which means that it /could/ change how people script against the login call.... I am ~10% confident about this one.

Reply:
Great point. Thank you for that good resource on that. Switching now.